### DISCUSSION TITLE
kubevela trigger能否跟钉钉预警结合

### DISCUSSION THREAD

**Discussion by haima95** [kubevela/kubevela#5858] [Q&A]
kubevela triger触发后的action可以设置为钉钉机器人发消息吗？如果可以，要怎么操作？ 很感谢能抽空解答

**Comment by charlie0129** [comment]
Currently, I'm afraid it is not possible with built-in actions. However, this will be supported in the future. If you are in a hurry, you can write your custom action to do this, just like the built-in ones https://github.com/kubevela/kube-trigger/blob/main/config/definition/create-event-listener.yaml . If you decide to write this action, I think it can also be merged into the `kube-trigger` repo to become the built-in ones.

**Comment by FogDong** [comment]
可以的。目前官方有几种内置的 action，你可以参考官方的 action 自定义自己的 action：https://github.com/kubevela/kube-trigger/blob/main/config/definition/bump-application-revision.yaml#L7

在自定义 action 中，你可以通过引用 `vela/http` 包，来使用 http 发送钉钉消息请求。
